http/headers/alt-svc.json

```diff
@@ -0,0 +1,105 @@
+{
+  "http": {
+    "headers": {
+      "Accept": {
```

Comment:
```suggestion
      "Alt-Svc": {
```

Reply:
We should really add tests to prevent mistakes like this.